libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceSDKIdentityTests.m

```diff
@@ -123,16 +123,17 @@ - (void)FIXMEtestRetrieveIdentityFailure
     idCoord.credentials.identityUrl = origIdentityUrl;
 }
 
-- (void)FIXMEtestIdentityAuthRefreshSuccess
+- (void)testIdentityAuthRefreshSuccess
 {
-    [SFAuthenticationManager sharedManager].idCoordinator.idData = nil;
-    [SFAuthenticationManager sharedManager].idCoordinator.credentials.accessToken = @"BadToken";
-    [self sendSyncIdentityRequest];
-    XCTAssertEqualObjects(_requestListener.returnStatus, kTestRequestStatusDidLoad, @"Identity request failed.");
+    SFAuthenticationManager *sharedManager = [SFAuthenticationManager sharedManager];
+    sharedManager.coordinator.credentials.accessToken = @"BadToken";
+    sharedManager.coordinator.credentials.instanceUrl = [NSURL URLWithString:@"https://cs2.salesforce.com"];
```

Comment:
Shouldn't this (and the comparison below) be based more dynamically on the configured credentials?  I'd suggest we:

1. Save off the original `instanceUrl`.
2. Set the "bad" instanceUrl to something it couldn't be (e.g. "https://www.example.com").
3. Make the comparison below against the saved instanceUrl.